python/ray/dashboard/modules/reporter/tests/test_gpu_profiler_manager.py

```diff
@@ -0,0 +1,333 @@
+"""Unit tests for the GPU profiler manager.
```

Comment:
Will add an e2e test with real GPU + dynolog in a follow-up. For now, I've tested e2e manually.